components/form-builder/app/responses/Dialogs/DownloadDialog.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect } from "react";
+import { Dialog, useDialogRef } from "../../shared";
+import { useTranslation } from "react-i18next";
+import { Button } from "@components/globals";
+import { logMessage } from "@lib/logger";
+import axios from "axios";
+import { DownloadFormat } from "@lib/responseDownloadFormats/types";
+import JSZip from "jszip";
+
+export const DownloadDialog = ({
+  checkedItems,
+  isVisible,
```

Comment:
Maybe rename this so it's more clean that this shows or hides the dialog.

Reply:
renamed to isDialogVisible, setIsDialogVisible